error TS2345: Argument of type '{ view: any; caption: string; }' is not assignable to parameter of type 'ITNSCoachMark'.   Object literal may only specify known properties, and 'view' does not exist in type 'ITNSCoachMark'.

Hello,
I am unable to integrate this plugin in my Nativescript angular project. Can you share some sample code for accessing this plugin. I tried but keep on getting this error:  
```
error TS2345: Argument of type '{ view: any; caption: string; }' is not assignable to parameter of type 'ITNSCoachMark'.
  Object literal may only specify known properties, and 'view' does not exist in type 'ITNSCoachMark'.
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -36,6 +36,7 @@ export interface ITNSCoachMark {
     labelPosition?: number;
     labelAlignment?: number;
     showArrow?: boolean;
+    view?: any;
 }
 export interface ICONTINUE_LOCATION {
     TOP: number;
@@ -49,6 +50,7 @@ export declare class TNSCoachMark {
     labelPosition: number;
     labelAlignment: number;
     showArrow: boolean;
+    view?: any;
     static SHAPES: ICoachMarkShape;
     static LABEL_POSITIONS: ICoachMarkLabelPosition;
     static LABEL_ALIGNMENTS: ICoachMarkLabelAlignment;

